examples/update-states.js

```diff
@@ -0,0 +1,89 @@
+const async = require('async');
+const Redis = require('ioredis');
+
+/*
+This script updates the state of Utapi `numberOfObjects` and `storageUtilized`
+starting from the current timestamp (i.e., the latest 15 minute interval).
+
+To use:
+* Set REDIS_ENDPOINT to the host and port where the Redis server is running
+* Set the various metric resource states (STATES) to your desired values. This
+  can be an array of n length of objects. Each value `storageUtilized` and
+  `numberOfObjects` of the state objects is optional, and you should only
+  include those that you want to change.
+*/
+
+const REDIS_ENDPOINT = {
+    host: '127.0.0.1',
+    port: 6379,
+};
+
+const STATES = [
+    {
+        resource: 'buckets', // required
+        bucket: 'test-bucket', // required
+        storageUtilized: '0',
+        numberOfObjects: '0',
+    },
+    {
+        resource: 'accounts', // required
+        accountId: '<account-canonical-id>', // required
+        storageUtilized: '0',
+        numberOfObjects: '0',
+    },
+    {
+        resource: 'users', // required
+        userId: '<user-id>', // required
+        storageUtilized: '0',
+        numberOfObjects: '0',
+    },
+    {
+        resource: 'service', // required
+        service: '<service-name>', // required
+        storageUtilized: '0',
+        numberOfObjects: '0',
+    },
+];
+
+function generateStateKey(params, metric) {
+    const { bucket, accountId, userId, service, resource } = params;
+    const id = bucket || accountId || userId || service;
+    return `s3:${resource}:${id}:${metric}`;
```

Comment:
Can you use the method in schema.js to generate this state key?

Reply:
I had done that originally, but thought it best to not be tied to any Utapi source code. That way users can take it and run it wherever they want without cloning Utapi, or using any particular Docker container.

Reply:
That makes sense, it is a good flexibility to have.